test/src/e2e_vm_tests/test_programs/should_fail/vec_set_index_out_of_bounds/Forc.toml

```diff
@@ -0,0 +1,8 @@
+[project]
```

Comment:
A `test.toml` is missing here. You can look at other `should_fail` `Vec` tests to see how to write one :)

Reply:
[Ref](https://github.com/FuelLabs/sway/blob/master/test/src/e2e_vm_tests/test_programs/should_fail/vec_swap_param1_out_of_bounds/test.toml)